components/provider_ppucd/Cargo.toml

```diff
@@ -0,0 +1,37 @@
+# This file is part of ICU4X. For terms of use, please see the file
+# called LICENSE at the top level of the ICU4X source tree
+# (online at: https://github.com/unicode-org/icu4x/blob/master/LICENSE ).
+
+[package]
+name = "icu_provider_ppucd"
+description = "Data provider that reads from a JSON formatted PPUCD data source"
+version = "0.1.0"
+authors = ["The ICU4X Project Developers"]
+edition = "2018"
+repository = "https://github.com/unicode-org/icu4x"
+license-file = "../../LICENSE"
+categories = ["internationalization"]
+include = [
+    "src/**/*",
+    "examples/**/*",
+    "benches/**/*",
+    "Cargo.toml",
+    "README.md"
+]
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+icu_locid = { version = "0.1", path = "../locid" }
+icu_provider = { version = "0.1", path = "../provider" }
+serde = { version = "1.0", features = ["derive"] }
+serde_json = "1.0"
```

Comment:
Why do you need serde_json in provider_ppucd?

Reply:
Done. Removed, I didn't need serde_json for PPUCD Provider, but didn't realize it originally, and thereafter hadn't removed the stale reference.